### PR TITLE
Update index.ts

### PR DESCRIPTION
**Pull request by ahobdaysuperthread**
Fixed some minor grammar issues, replaced apostrophes with "smart" apostrophes, and shortened messages where I could to save space.

### CODE REVIEW

File: src/en/index.ts [hunk 1]
```diff
@@ -773,14 +773,14 @@ const en = {
...
-    "This could be your first name, or a nickname — however you'd like people to refer to you in Superthread.",
+    "This could be your first name, or a nickname. It will be displayed around the app.",
   embedVideo: "Embed Youtube video",
   removePullRequest: "Remove pull request",
   viewOnGitHub: "View on GitHub",
   setUpGHIntegration: "Set up GitHub integration",
   integrateSuperthread: "Integrate Superthread into your favorite git client.",
   githubIntegradionDescripiton:
-    "Link Superthread cards to GitHub pull requests. Include a Superthread Card-ID (e.g ST-123) in your branch name/s or pull request title/s to automatically show the statuses of the PRs in the Card.",
+    "Link Superthread cards to GitHub pull requests. Include a Superthread card ID (e.g C-123) in your branch name or pull request title to automatically show the PR status in the card.",
```
Comment (aldinezi) [inline src/en/index.ts:783]: @ben-bourdin451 I think github integration will work only if the branch name includes ST-${cardId}. Will it work with C- ?

Comment (ben-bourdin451) [reply src/en/index.ts:783]: yes it also should work with `c-` prefix

**Review by aldinezi — APPROVED** [review]

**Review by aldinezi — CHANGES_REQUESTED** [review]
please run 
`npm install` 
and then
`npm run format`

File: src/en/index.ts (hunk 1 above)
Comment (ahobdaysuperthread) [reply src/en/index.ts:783]: Ah, I misunderstood this message, and thought it had been written before Superthread had settled on the "C" prefix for cards. Happy to change this back if it makes things more clear.

**Comment by ahobdaysuperthread** [comment]
> please run `npm install` and then `npm run format`

@aldinezi Is this directed at me? I edited the file in GitHub, so I assume anything I run on my computer won't help with whatever formatting issue I've caused.

**Comment by aldinezi** [comment]
> > please run `npm install` and then `npm run format`
> 
> @aldinezi Is this directed at me? I edited the file in GitHub, so I assume anything I run on my computer won't help with whatever formatting issue I've caused.

yes @ahobdaysuperthread 
you would need to clone the repo to your computer, 
checkout to your branch `ahobdaysuperthread:patch-1`, 
install dependencies with (npm install), and then run `npm format`. 
then you would need to commit the changes and push to your PR.

**Comment by ahobdaysuperthread** [comment]
> yes @ahobdaysuperthread you would need to clone the repo to your computer, checkout to your branch `ahobdaysuperthread:patch-1`, install dependencies with (npm install), and then run `npm format`. then you would need to commit the changes and push to your PR.

@aldinezi I'd like to make changes to this file directly in GitHub. Is there a practical way for me to do that? e.g. are the rules for formatting written down somewhere, so I can spot what I might have changed that caused this issue?

**Comment by aldinezi** [comment]
> > yes @ahobdaysuperthread you would need to clone the repo to your computer, checkout to your branch `ahobdaysuperthread:patch-1`, install dependencies with (npm install), and then run `npm format`. then you would need to commit the changes and push to your PR.
> 
> @aldinezi I'd like to make changes to this file directly in GitHub. Is there a practical way for me to do that? e.g. are the rules for formatting written down somewhere, so I can spot what I might have changed that caused this issue?

you could technically do it manually. but the ruleset is defined in `prettier` library, and it fixes your errors automatically on save in vscode editor, as this thing was not meant to be edited via github GUI, it is fine for small changes though.

let me pull your branch and fix it for you.

**Review by aldinezi — APPROVED** [review]

**Pull request merged**